app/commands/.lingo

```diff
@@ -1,9 +1,12 @@
-lexicons:
-- codelingo/common as _
 tenets:
-- name: find-funcs
-  doc: Example tenet that finds all functions.
-  comment: This is a function, but you probably already knew that.
-  match: |2
-      <func
+  - name: find-funcs
```

Comment:
Remove this tenet completely. Think about what would happen if you run a review in this repo with this tenet?